bitcoin/src/lib.rs

```diff
@@ -150,12 +150,9 @@ impl LockedTransaction {
 
 fn get_exponential_backoff() -> ExponentialBackoff {
     ExponentialBackoff {
-        max_elapsed_time: Some(Duration::from_secs(24 * 60 * 60)),
-        max_interval: Duration::from_secs(5 * 60), // wait at most 5 minutes before retrying
-        initial_interval: Duration::from_secs(1),
-        current_interval: Duration::from_secs(1),
```

Comment:
Why did you remove this? Imo the defaults are not suitable for us - it will retry a ton of times in the first few seconds. And also I prefer to have the configuration explicit.